ci/do_ci.sh

```diff
@@ -203,9 +203,14 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
   setup_clang_libcxx_toolchain
   # This doesn't go into CI but is available for developer convenience.
   echo "bazel with different compiletime options build with tests..."
+
+  if [[ "${TEST_TARGETS}" == "//test/..." ]]; then
+    cd "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
+    TEST_TARGETS="@envoy//test/..."
```

Comment:
This seems a bit special casey..

Reply:
This is for the default case (where TEST_TARGETS is not specified), and for CI setup.